### ISSUE TITLE
toggle increases size

### ISSUE THREAD

**Issue by jetic83** [picturae/OpenSeadragonMagnifier#5]
Each time the m time toggles the magnifier on, the magnified area gets larger and larger. This is obviously a bug?!

**Comment by MindFreeze** [comment]
Definitely. Though I haven't tried to reproduce and have no idea when it is going to be fixed.

**Comment by jetic83** [comment]
It's easy to reproduce: in the demo, toggle on the magnifier, then zoom into the magnifier such that you see a small red rectangle on the main image.
Then toggle on and off the magnifier with 'm' and the red rectangle will get larger and larger.

**Comment by jetic83** [comment]
it can be sort of solved if after toggleVisibility() the function updateSize() is called.

**Comment by rster2002** [comment]
Hi! From what I have tested this seems to have been fixed somewhere during the years so I'm closing this issue.
